ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -171,20 +171,30 @@ SVFrequecyHeaderFilter.propTypes = {
   onChange: PropTypes.func,
 }
 
-const QS_FILTER_FIELD = {
+const SV_QS_FILTER_FIELD = {
   name: 'min_qs',
-  label: 'SV Quality Score',
+  label: 'Exome SV Quality Score',
   labelHelp: (
     <span>The quality score (QS) represents the quality of a Structural Variant call. Recommended SV-QS cutoffs for filtering:<br />
-      WGS: &gt; 10; <br />
-      WES: duplication &gt;= 50, deletion &gt;= 100, homozygous deletion &gt;= 400.
+      duplication &gt;= 50, deletion &gt;= 100, homozygous deletion &gt;= 400.
```

Comment:
we no longer need the `<br />` and new lines, which means that for both sv quality fields the `labelHelp` can just be a plain string instead of HTML